lightkurve/search.py

```diff
@@ -667,6 +676,7 @@ def _query_mast(target, radius=None, project=['Kepler', 'K2', 'TESS']):
                 # suppress misleading AstropyWarning
                 warnings.simplefilter('ignore', AstropyWarning)
                 from astroquery.mast import Observations
+                log.debug("Started querying MAST for coordinates='{} {}'.".format(ra, dec))
```

Comment:
This may be confusing if a user enters a `target` name and gets a message about querying coordinates... Since this only happens during a cone search it might make sense to specify 
```python
log.debug("Started querying MAST for observations within {} arcsec "
          "of target coordinates='{} {}'.".format(radius.to(u.arcsec), ra, dec))
```

If this is getting too wordy, don't worry about it. Just a thought!

Reply:
I like it!  I'm going to add the `radius` to all debug messages in search, because I think it's always being used in the query.